images/runtime/php-fpm/8.1/base.bullseye.Dockerfile

```diff
@@ -93,7 +90,7 @@ RUN set -eux; \
 #  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
 #  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
 RUN set -eux; \
-    if [[ $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.0.* ]]; then \
+    if [[ $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.* ]]; then \
```

Comment:
The only PHP versions we have right now are 7.4, 8.0, and 8.1 right? So is there a reason we don't just remove this IF statement?